src/pages/index.js

```diff
@@ -7,7 +7,6 @@ import Layout from '../components/Layout';
 import useFetch from '../hooks/useFetch';
 
 export default function HomePage() {
-
 	const {data, error, loading} = useFetch('api/teaser');
```

Comment:
You should use loading if you initialize the variable.

Reply:
deleted loading